gen/gen.go

```diff
@@ -44,10 +44,11 @@ func init() {
 	}
 }
 
-func Gen(jFile string, j interface{}, t *template.Template) error {
+func Gen(exercise string, j interface{}, t *template.Template) error {
 	if dirMetadata == "" {
 		return errors.New("unable to determine current path")
 	}
+	jFile := filepath.Join("exercises", exercise, "canonical-data.json")
```

Comment:
I'm not sure how idiomatic this is. Is it worth moving the "exercises" and "canonical-data.json" strings into variables within the function or even package level so they're further up the file, making them more obvious and potentially easier to change in the future?? or not??